max.rb

```diff
@@ -0,0 +1,80 @@
+# Method name: max
```

Comment:
Awesome!  I like that you added `max.rb` as a separate file in order to make use of your own `max` method.  That's one of the major things we like to see students start doing. :smile: 

Now that you've done it effectively, I want to draw your attention to Ruby's [Enumerable#max](http://ruby-doc.org/core-2.2.0/Enumerable.html#method-i-max) method.  Any `Enumerable` object in Ruby — which includes `Arrays` and `Hashes` — has a `max` method automatically.  This means if you have an array you can do

```ruby
array = [10, 30, -15, 14]
array.max # => 30
```